### PR TITLE
add parse docker stats to support job exporter

### PR DESCRIPTION
**Pull request by YanjieGao**
parse docker stats cmd and extract the info for later job exporter to use

### CODE REVIEW

File: prometheus/exporter/docker_stats.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    # pop the headers
+    data.pop(0)
+    rowNum = len(data)
+    colNum = len(data[0])
+    containerStats = {}
+
+    for i in range(rowNum):
+        id = data[i][1]
+        containerInfo = {
+            "Container": data[i][0],
```
Comment (fanyangCS) [inline prometheus/exporter/docker_stats.py]: >Container [](start = 13, length = 9)

what does this mean? container name?

Comment (YanjieGao) [reply prometheus/exporter/docker_stats.py]: dockerStatsCMD = "sudo docker stats --no-stream --format \"table {{.Container}}, {{.ID}}, {{.CPUPerc}},{{.MemUsage}},{{.NetIO}},{{.BlockIO}},{{.MemPerc}}\""

query results example:
CONTAINER,CONTAINER ID,CPU %,MEM USAGE / LIMIT,NET I/O,BLOCK I/O,MEM %
33a22dcd4ba3,33a22dcd4ba31ebc4a19fae865ee62285b6fae98a6ab72d2bc65e41cdc70e419,0.00%,18.68MiB / 300MiB,0B / 0B,28MB / 0B,6.23%

container is the short container id which will show at cmd results

File: prometheus/exporter/docker_stats.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            "MemUsage_Limit": parseUsageLimit(data[i][2]),
+            "NetIO": parseIO(data[i][3]),
+            "BlockIO": parseIO(data[i][4]),
+            "MemPerc": parsePercentile(data[i][5])
+        }
+        containerStats[id] = containerInfo
+    return containerStats
+    
+def stats(argv):
+    print(argv)
```
[the remark this answers is not in the document]
Comment (fanyangCS) [reply prometheus/exporter/docker_stats.py]: Yanjie might consider add logDir in argv later?

---
In reply to: [180317755](https://github.com/Microsoft/pai/pull/448#discussion_r180317755) [](ancestors = 180317755)

[the remark this answers is not in the document]
Comment (YanjieGao) [reply prometheus/exporter/docker_stats.py]: removed

[the remark this answers is not in the document]
Comment (sterowang) [reply prometheus/exporter/docker_stats.py]: If so, suggest to add specific parameter later.

---
In reply to: [180319835](https://github.com/Microsoft/pai/pull/448#discussion_r180319835) [](ancestors = 180319835,180317755)

[the remark this answers is not in the document]
Comment (YanjieGao) [reply prometheus/exporter/docker_stats.py]: remove this, will move it to job exporter file

**Pull request merged**